config/locales/en.yml

```diff
@@ -89,56 +87,63 @@ en:
         title: Pending opportunities
   opportunity_status:
     not_approved: If everything looks good, submit this opportunity for approval.
-    pending_approval: This opportunity won't go live until you approve it.
+    pending_approval: Vendors can't see this opportunity yet. It will only be published after you approve it.
     posted: This opportunity is posted and viewable to the public.
     submitted_for_approval: You submitted this opportunity for approval on %{timestamp}.
     waiting_for_publish_date: This opportunity will be posted on %{timestamp}.
   pending_opportunities: Pending opportunities
   phone: Phone
-  post_opportunity: Post opportunity
-  posted_time_ago: Posted %{time_ago}
+  post_an_opportunity: Post an opportunity
+  public_contact_info: Public contact information
+  publish_your_answer: Publish your answer
   question_text: Question text
   questions: Questions
   request_approval: Request approval
-  rss_for_search: RSS Feed for this search
+  rss: RSS
+  save_changes: Save changes
   save_and_continue: Save and continue
-  save_search: Save this search
-  saved_searches: Saved searches
-  search_saved: This search has been saved. We'll email you with any new results.
-  search_saved_at_time: Search saved on %{timestamp}
+  search_for_opportunities: Search for opportunities
+  filter_saved: We'll email you with any new results for this filter. You can unsubscribe by clicking your name in the top navigation bar.
+  filter_subscribed_at_time: Filter subscribed to on %{timestamp}
   sign_in: Sign in
   sign_out: Sign out
   simple_form:
     hints:
       opportunity:
         category_ids: |-
-          Vendors who subscribe to these categories will be notified of your opportunity.
-          Choose these carefully!
-        contact_info: This information will be publicly displayed on your opportunity page
+          Your choice of categories will affect the vendors that see your opportunity.
```

Comment:
This seems to infer that vendors can only see opportunities that are in their category.

How about s/see/are notified of/?